src/pydocstyle/config.py

```diff
@@ -208,7 +208,10 @@ def _get_config(self, node):
         if path in self._cache:
             return self._cache[path]
 
-        config_file = self._get_config_file_in_folder(path)
+        if self._options.config_file:
```

Comment:
I think letting `--config` also search up will be of no use to anyone. I can see how `--config` could be used to, for example, point to a shared config file that is not in any parent directory, but just using a different project's specific directory's configuration... is at least rare, if not meaningless. I suggest the following: 

* Make `--config` take a single, specific file. 
* Using `--config` will cancel any other configuration file search, and inheritance (i.e., it will imply `inherit = false`).
* For the case where we want to "flatten" a complex inherited configuration to a single file, we can add a flag or a script that ignores actual files and just print the computed configuration.

Thoughts?